### PR TITLE
Refactor schemas to add Actions and Forms

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

**Comment by tshemsedinov** [comment]
@nechaido here is Action example. I propose to get category properties marked with `Action` decorator and insert them to `Action` table after all schemas import.

**Comment by nechaido** [comment]
@tshemsedinov how will we define arguments and their type for an Action, that does not have a Form associated with it?

IIRC there should be `Arguments` decorator.

**Comment by tshemsedinov** [comment]
Waiting for https://github.com/metarhia/globalstorage/pull/262 @nechaido

**Comment by tshemsedinov** [comment]
ping @nechaido @belochub

File: schemas/system/SignUp.schema [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Form({
+  Login: { domain: 'Login', required: true },
+  Password1: { domain: 'PasswordHash', required: true },
```
Comment (nechaido) [inline schemas/system/SignUp.schema]: IIUC this would be just a string, it will become hash when we save a password to a db.

Comment (nechaido) [reply schemas/system/SignUp.schema]: I'd prefer to name field `FirstPassword` and `SecondPassword` accordingly.

Comment (tshemsedinov) [reply schemas/system/SignUp.schema]: We may use SHA-512
@belochub ?

Comment (belochub) [reply schemas/system/SignUp.schema]: I think domain should be called something like `PlainTextPassword`.

**Comment by tshemsedinov** [comment]
Landed in 452266e and 8e73d63

**Pull request closed without merging**